### ISSUE TITLE
Provided demo code never outputs anything to the console.log()

### ISSUE THREAD

**Issue by budda** [driftyco/ionic-ion-tinder-cards#79]
No matter what events are used on-transition-left vs on-swipe-left etc. none ever get called to output the demo debug text to the console.log

Looking at the code i'm assuming as cards are swiped off new cards should be added? This too doesn't happen so I eventually get to the end of the card pile.

I'm testing in the Chrome browser. But even on iOS with Ionic View I can get to the end of he card pile as no new cards are being added.

The only output I do see in the console is regarding:

```
'webkitMovementX' is deprecated. Please use 'movementX' instead. ionic.bundle.js:1162
'webkitMovementY' is deprecated. Please use 'movementY' instead. ionic.bundle.js:1162
```

**Comment by keithdmoore** [comment]
This may be due to a new version of Ionic.

**Comment by karladler** [comment]
I''m not sure how is it supposed to work, but I guess it's because "cardSwipedLeft" and "cardSwipedRight" are called outside the "CardCtrl" Controller. When I move them into the "CardsCtrl" everything works fine.

**Comment by budda** [comment]
@abimelex you are completely correct with your observation and all event output to console works by moving them to the parent controller.
